docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -0,0 +1,69 @@
+// tag::ess[]
+
+// QUESTION: Should we add a warning here about updating to 7.13 and include a
+// link to the upgrade docs? 
+
+{ecloud} runs a hosted version of {fleet-server}. To use {fleet-server}, start
+the service:
+
+//TODO: Describe how to start the service.
+
+When you're done, log in to {kib} and go to *Management > {fleet}*. Click the
+*Agents* tab to confirm that {fleet-server} is running and healthy.
+
+//TODO: Add screen capture.
+
+On the *Policies* tab, notice that the {agent} is enrolled in the Elastic Cloud
```

Comment:
```suggestion
On the *Policies* tab, notice that the {agent} is enrolled in the 'Elastic Agent on Cloud policy'
```

Reply:
<img width="491" alt="Screen Shot 2021-04-26 at 6 33 36 PM" src="https://user-images.githubusercontent.com/12970373/116159295-f981df80-a6bd-11eb-9880-e548addd6dba.png">

Reply:
and if there is a better way to cite the policy's actual name, go for it, I don't know if it is proper to use 'single' quotes or if we should just use Captilization, or what.  :)

Reply:
Typically we do not put names in quotes like this.